### PR TITLE
Implement ItemSelectedEventArgsConverter in converters

### PR DESCRIPTION
**Pull request by almirvuk**
### Description of Change ###

This is a very simple converter that allows you to extract `SelectedItem` value from `ItemSelectedEventArgs` object and it can be used in combination with `EventToCommandBehavior`.

Added: 
 
- `ItemSelectedEventArgsConverter`

### Sample

```
<c:ItemSelectedEventArgsConverter x:Key="ItemSelectedEventArgsConverter" />

<ListView.Behaviors>
      <b:EventToCommandBehavior EventName="ItemSelected" 
                                Command="{Binding ItemSelectedCommand}"
                                EventArgsConverter="{StaticResource ItemSelectedArgsConverter}" />
</ListView.Behaviors>   
```

### PR Checklist ###

- [x] Has tests (if omitted, state reason in description)
- [x] Has samples (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [ ] Updated documentation

### CODE REVIEW

File: XamarinCommunityToolkit.UnitTests/Converters/ItemSelectedEventArgsConverter_Tests.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+using System.Globalization;
+using Xamarin.Forms;
+using System;
+using Xunit;
+
+namespace XamarinCommunityToolkit.UnitTests.Converters
+{
+    public class ItemSelectedEventArgsConverter_Tests
+    {
+        public static object expectedValue = 100;
```
Comment (jfversluis) [inline XamarinCommunityToolkit.UnitTests/Converters/ItemSelectedEventArgsConverter_Tests.cs:12]:
```suggestion
       static object expectedValue = 100;
```

This doesn't need to be public, right? It's a nit, but just to make sure people aren't finding it with IntelliSense or something :)

File: XamarinCommunityToolkit.UnitTests/Converters/ItemSelectedEventArgsConverter_Tests.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        {
+            var itemSelectedEventArgsConverter = new ItemSelectedEventArgsConverter();
+
+            var result = itemSelectedEventArgsConverter.Convert(value, typeof(ItemSelectedEventArgsConverter), null, CultureInfo.CurrentCulture);
+            Assert.Equal(result, expectedResult);
+        }
+
+        [Theory]
+        [InlineData("Random String")]
+        public void InValidConverterValuesThrowArgumenException(object value)
```
Comment (jfversluis) [inline XamarinCommunityToolkit.UnitTests/Converters/ItemSelectedEventArgsConverter_Tests.cs:31]:
```suggestion
        public void InvalidConverterValuesThrowsArgumentException(object value)
```

File: XamarinCommunityToolkit.UnitTests/Converters/ItemSelectedEventArgsConverter_Tests.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        public static object expectedValue = 100;
+
+        public static IEnumerable<object[]> GetData() => new List<object[]>
+        {
+            new object[] { new SelectedItemChangedEventArgs(expectedValue), expectedValue},
+        };
+
+        [Theory]
+        [MemberData(nameof(GetData))]
+        public void InverterBoolConverter(SelectedItemChangedEventArgs value, object expectedResult)
```
Comment (jfversluis) [inline XamarinCommunityToolkit.UnitTests/Converters/ItemSelectedEventArgsConverter_Tests.cs:21]: This is not a correct method name? Too much copy/paste? ;)

Comment (almirvuk) [reply XamarinCommunityToolkit.UnitTests/Converters/ItemSelectedEventArgsConverter_Tests.cs:21]: I'm guilty 😂

**Pull request merged**